x-pack/plugins/observability_solution/observability/server/plugin.ts

```diff
@@ -146,7 +146,7 @@ export class ObservabilityPlugin implements Plugin<ObservabilityPluginSetup> {
           all: {
             app: [observabilityFeatureId],
             catalogue: [observabilityFeatureId],
-            api: ['rac'],
```

Comment:
eh, does this mean they never worked before?

Reply:
Earlier we didn't have any access privilege specified for this API. So it worked before too as far as I know.

Reply:
hmm I actually thought `tags` were used for this purpose, but I'm not sure if we tested it in any case. Do you mind adding an API test that checks if there's a 403 when using a user with insufficient access privileges?

Reply:
oh, now I get it, this is for the Observability plugin. Hmm. I think the outcome of this should be that you only have access to this API if you have the Obs AI Assistant feature privilege. With this change, is that the case?

Reply:
Sorry, what I mean is that the feature privilege for the AI Assistant is defined here:

https://github.com/elastic/kibana/blob/c46d94ca8d5f783ace821064e1482d2fbdc6ab7c/x-pack/plugins/observability_solution/observability_ai_assistant/server/plugin.ts#L80

If the user has this privilege, they should have access to `/internal/observability/assistant/alert_details_contextual_insights`. If they don't have this privilege, they shouldn't. If I understand this PR correctly, you're changing a different feature privilege?

Reply:
I'm adding the same feature privilege (`ai_assistant`) to the contextual details endpoint. With the new authz migration, `options.tags` are not used anymore. It's added under the `security.authz.requiredPrivileges` property.